### PR TITLE
Community Broadcast page updates

### PR DESCRIPTION
**Pull request by kylefraser**
This PR updates the layout of the Community Broadcast page.

These changes were initially discussed on [Slack]( https://trinodb.slack.com/archives/C01GW0QR16Z/p1613595522041100) and were further materialized via discussion with @bitsondatadev 

One of the goals is to highlight more of the podcast content, as opposed to having to click the current "See Past Episodes" button.

Included:
- Call out 'Latest Episode' and embed video
- Grid/Blog style for previous episodes
- Watch/Listen links are now icons
- Google Calendar now just displays the 'Agenda' 
- Previous episode cards are limited to 3 bulleted items

<details>
<summary>screenshot</summary>

screencapture-127-0-0-1-4000-broadcast-2021-02-25-10_53_55

</details>

Related Slack Thread: https://trinodb.slack.com/archives/C01GW0QR16Z/p1613595522041100

### CODE REVIEW

File: broadcast/index.md [hunk 1]
```diff
@@ -6,10 +6,10 @@ show_hero: true
...
 <div class="container container__broadcast">
 
-  <div class="row spacer-60">
-    <div class="col-md-12">
-
-<div markdown="1" class="leftcol widecol">
+<div class="row spacer-60">
+<div class="d-flex justify-content-between flex-column flex-lg-row" style="margin-bottom: 1rem;">
+<div markdown="1" class="col-lg-8">
+## About
```
Comment (mosabua) [inline broadcast/index.md:12]: remove .. this is redundant

File: broadcast/index.md [hunk 2]
```diff
@@ -24,28 +24,91 @@ and let us know there. Otherwise, you can message Manfred Moser or Brian Olsen
...
-
+<div markdown="1" class="col-lg-3">
 ## Co-hosts
 
- - Brian Olsen [@bitsondatadev](https://twitter.com/bitsondatadev)
- - Manfred Moser [@simpligility](https://twitter.com/simpligility)
+- Brian Olsen [@bitsondatadev](https://twitter.com/bitsondatadev)
+- Manfred Moser [@simpligility](https://twitter.com/simpligility)
+
+## Watch/Listen
```
Comment (mosabua) [inline broadcast/index.md:35]: Watch or listen

File: broadcast/index.md [hunk 3]
```diff
@@ -24,28 +24,91 @@ and let us know there. Otherwise, you can message Manfred Moser or Brian Olsen
...
+- Listen:
+<a href="https://podcasts.apple.com/us/podcast/trino-community-broadcast/id1533484786" target="_blank">
+  <i class="fab fa-apple watch-listen-icon" title="Apple"></i>
+</a>
+<a href="https://podcasts.google.com/feed/aHR0cHM6Ly9mZWVkcy5idXp6c3Byb3V0LmNvbS8xMzc0NTMyLnJzcw==" target="_blank">
+  <i class="fab fa-google watch-listen-icon" title="Google"></i>
+</a>
+<a href="https://open.spotify.com/show/53ZrVCCmZSsEmvlNfzpWSL" target="_blank">
+  <i class="fab fa-spotify watch-listen-icon" title="Spotify"></i>
+</a>
```
Comment (mosabua) [inline broadcast/index.md:52]: Do we need to do a generic RSS feed or something like that a well.. something that works for any podcast app

File: broadcast/index.md [hunk 4]
```diff
@@ -24,28 +24,91 @@ and let us know there. Otherwise, you can message Manfred Moser or Brian Olsen
...
+</div>
+</div>
+<div markdown="1" class="col-md-12">
+## Upcoming shows
```
Comment (mosabua) [inline broadcast/index.md:56]: empty line below and above title

File: broadcast/index.md [hunk 5]
```diff
@@ -24,28 +24,91 @@ and let us know there. Otherwise, you can message Manfred Moser or Brian Olsen
...
 
-## Where to watch or listen
+<iframe src="https://calendar.google.com/calendar/embed?height=600&amp;wkst=1&amp;bgcolor=%23dd00a1&amp;ctz=America%2FDetroit&amp;src=NDhibXJvaXVpZmg2NWJsZWNhOGxzNGhyNTRAZ3JvdXAuY2FsZW5kYXIuZ29vZ2xlLmNvbQ&amp;color=%23D81B60&amp;mode=AGENDA&amp;showCalendars=0&amp;showTz=1&amp;showTabs=0&amp;showPrint=0" style="border:solid 1px #777; margin-bottom: 2rem;" width="100%" height="240" frameborder="0" scrolling="no"></iframe>
 
- - Watch live on <a href="https://www.twitch.tv/trinodb" target="_blank">Trino Twitch</a>
- - Watch later on <a href="https://www.youtube.com/playlist?list=PLFnr63che7war_NzC7CJQjFuUKLYC7nYh" target="_blank">Youtube playlist</a>
- - Listen on <a href="https://podcasts.apple.com/us/podcast/trino-community-broadcast/id1533484786" target="_blank">Apple podcast</a>
- - Listen on <a href="https://podcasts.google.com/feed/aHR0cHM6Ly9mZWVkcy5idXp6c3Byb3V0LmNvbS8xMzc0NTMyLnJzcw==" target="_blank">Google podcast</a>
- - Listen on <a href="https://open.spotify.com/show/53ZrVCCmZSsEmvlNfzpWSL" target="_blank">Spotify</a>
+## Latest Episode
```
Comment (mosabua) [inline broadcast/index.md:60]: Latest episode

and empty line above and below title

File: broadcast/index.md [hunk 6]
```diff
@@ -24,28 +24,91 @@ and let us know there. Otherwise, you can message Manfred Moser or Brian Olsen
...
+        <div class="latest-entry-video-wrapper">
+          <iframe src="https://www.youtube.com/embed/{{ latestEpisode.youtube_id }}" frameborder="0" allowfullscreen></iframe>
+        </div>
+    </div>
+  </div>
+</div>
 
-<iframe src="https://calendar.google.com/calendar/embed?height=600&amp;wkst=1&amp;bgcolor=%23dd00a1&amp;ctz=America%2FDetroit&amp;src=NDhibXJvaXVpZmg2NWJsZWNhOGxzNGhyNTRAZ3JvdXAuY2FsZW5kYXIuZ29vZ2xlLmNvbQ&amp;color=%23D81B60&amp;mode=MONTH&amp;showCalendars=0&amp;showTz=1&amp;showTabs=1&amp;showPrint=0" style="border:solid 1px #777" width="100%" height="600" frameborder="0" scrolling="no"></iframe>
+<div markdown="1" class="previous-episode-header">
+## Previous Episodes
```
Comment (mosabua) [inline broadcast/index.md:89]: Empty line above and below

Previous episodes

**Review by mosabua — CHANGES_REQUESTED** [review]
A few tweaks needed but then good to go as a definite improvement!

**Review by bitsondatadev — APPROVED** [review]
Spoke with Kyle directly about fixing the ordering of "previous episodes." Otherwise looks fantastic!

**Pull request closed without merging**